demos/Demo.ts

```diff
@@ -145,7 +145,7 @@ class Demo {
         }
 
         function checkoutRedirect(txRequest: CheckoutRequest) {
-            return client.checkout(txRequest, new RedirectRequestBehavior());
+            return client.checkout(txRequest, new RedirectRequestBehavior(null, {storedGreeting: 'Hello Nimiq!'}));
```

Comment:
```suggestion
            return client.checkout(txRequest, new RedirectRequestBehavior(null, { storedGreeting: 'Hello Nimiq!' }));
```
:)

Reply:
to further simplify the API, one could also check if the first parameter is an object and thus assume = (null, state); But I think we hardly do any of this "parameter magic", right?

Reply:
Yeah, I don' think we do that anywhere. But if it becomes a common pain point, we can introduce it in the future.